### ISSUE TITLE
I am getting the same results with a simple UNet and FCN8

### ISSUE THREAD

**Issue by IssamLaradji** [DengPingFan/Inf-Net#14]
I used an FCN8  and UNet with an imagenet pretrained backbone and I got the same results as the proposed architecture (see Figure below). The test score is the DSC score on the test set.

In Table 2, did you compare against a UNet that (1) was pretrained on Imagenet and (2) uses the `joint_loss` in `Inf-Net/MyTrain_LungInf.py`? 
![image](https://user-images.githubusercontent.com/3382128/82755518-07d6c580-9da2-11ea-9d9d-5366b477d963.png)

**Comment by DengPingFan** [comment]
Dr.  IssamLaradji 
Sorry for the late response to your question.  In our experiment setting, we use the default setting (default training loss and pre-trained parameters) of the UNet (https://github.com/MrGiovanni/UNetPlusPlus) and FCN8. We have described and provided those links in our manuscript on page 8 (Sec. B. Experimental Settings.). We do not use our joint_loss for those models. 

From your new experiments, we can also conclude that our InfNet will converge quickly (about 15 minutes in our manuscript). Also, note that our Inf-Net is only a part of our semi-InfNet model which provides a new idea about our semi-supervised training strategy for a small training sample issue. 

If you have any other questions, please don't hesitate to contact me (dengpfan@gmail.com).

Best, Deng-Ping

**Comment by DengPingFan** [comment]
In addition, from your experiments, FCN8s is comparable to our InfNet model, partially due to our training data only 50 images. However, if you increase your training data to thousands of images. We believe the performance will be very different.
